api/reservations_api.py

```diff
@@ -122,6 +135,19 @@ class ReservationViewSet(viewsets.ModelViewSet):
     def perform_create(self, serializer):
         serializer.save(user_id=self.request.user.pk)
 
+    def get_queryset(self):
+        queryset = super().get_queryset()
+        user = self.request.user
+        return queryset.filter(
```

Comment:
Hmm, do we need to restrict visibility of reservations. Those are needed for showing availability of reservation unit. Although i guess this api/viewset shows information that shouldn't be visible there and probably needs a separate api endpoint.

Reply:
There is a task in JIRA about availablility of reservation unit from API and I was thinking when creating it that maybe we could retrieve availability in some different way than as list of reservations. In Respa we used reservation objects but with most of the information taken out of it for privacy reasons. 

However I thought at some point that it wasn't the best way to handle possibly hundreds of reservation objects and their possible buffer periods or some other things that block reserving for certain period. But I'm not sure if there is better way and you could very well be right about this.

However some information in reservations could be very private and shouldn't be showed to everyone. We don't have an implementation to filter that kind of information from public yet so because of that I made reservation visible for users themselves and admins etc only. 

I can of course change this for now because looks like there is not much private information in reservations yet.

Reply:
Probably leave as is, since we want to get the availability information from some place else anyway eventually, this kind of forces/reminds to implement that instead of relying on this.

Reply:
Leaving as is